src/databases/Silo/avtSiloFileFormat.C

```diff
@@ -10387,7 +10387,80 @@ avtSiloFileFormat::ReadInConnectivity(vtkUnstructuredGrid *ugrid,
                     vtk_zonetype != -1)
                 {
                     *nl++ = shapesize;
-                    for (k = 0 ; k < (size_t)shapesize ; k++)
+                    int nblinnod = 0 ;
+#if SILO_VERSION_GE(4,12,0)
```

Comment:
An alternative to using the `GE` macro here is to use `#ifdef DB_ZONETYPE_QUAD_BEAM` which I think is generally a little safer especially when coding for *future* releases.

Reply:
We use definition of `DB_ZONETYPE_QUAD_BEAM` as a proxy for when quadratic element support went into Silo.